core/src/main/java/org/apache/accumulo/core/clientImpl/ServerClient.java

```diff
@@ -88,6 +88,9 @@ public static <T> T executeRaw(ClientContext context,
   public static <CT extends TServiceClient,RT> RT executeRaw(ClientContext context,
       TServiceClientFactory<CT> factory, ClientExecReturn<RT,CT> exec) throws Exception {
     while (true) {
+      if (Thread.currentThread().isInterrupted()) {
```

Comment:
I made a comment with an alternate solution, but I think you want to call `Thread.interrupted` here as the Threads' interrupted state will be reset (to false) if it's true. https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/Thread.html#interrupted()